samples/notebooks/image-captioning-gcp/Image Captioning TF 2.0.ipynb

```diff
@@ -0,0 +1,893 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h1> Image Captioning with Attention in Tensorflow 2.0 </h1>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook modifies the [Image Captioning with Attention Tensorflow 2.0 notebook](https://colab.sandbox.google.com/github/tensorflow/docs/blob/master/site/en/r2/tutorials/text/image_captioning.ipynb)\n",
+    "to work with kubeflow pipelines."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h3> Download dataset and upload to GCS </h3>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we have to download the [MS COCO dataset](http://cocodataset.org/#download).  This sample uses both the 2014 train images and 2014 train/val annotations.  If you downloaded and extracted the dataset on your local system, you can upload it to GCS using `gsutil -m cp -r path/to/dataset/ gs://[YOUR-BUCKET-NAME]/ms-coco`."
```

Comment:
Does this sample use all the data or a subset? since the data size is huge, requiring users to download subset encourages users to try this sample out. I've spent 1-hour in downloading/uploading/fixing the data to the GCS.

Reply:
Now we download a subset of the dataset using gsutil commands at the beginning of the notebook.  The user should now be able to only run the cells and it will work with no previous set up (except for setting up the notebook server).

Reply:
Thanks. This is great.